debian/nnstreamer.install

```diff
@@ -4,6 +4,6 @@
 /usr/lib/nnstreamer/decoders/libnnstreamer_decoder_image_labeling.so
 /usr/lib/nnstreamer/decoders/libnnstreamer_decoder_direct_video.so
 /usr/lib/nnstreamer/filters/libnnstreamer_filter_cpp.so
-/usr/lib/*/gstreamer-1.0/*.so
+/usr/lib/*/gstreamer-1.0/libnnstreamer.so
```

Comment:
As we discussed in offline, I think it needs a quick patch because the install of grpc plugin is being blocked.
Let's review & merge this PR as soon as possible, or separate the PR.

Reply:
I'm not sure how long it will take to merge, so I will separate the PR related to this.